client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -30,6 +30,7 @@ export default function Dashboard(): JSX.Element {
   return (
     <Grid container component="main" className={`${classes.root} ${classes.dashboard}`}>
       <CssBaseline />
+      <Redirect to={{ pathname: '/search' }} />
```

Comment:
Just to get on the same page, this essentially makes this component unused. Did we intend this?